woocommerce/woocommerce-android
# Hackweek project - Optional leakcanary installation

### Description

The PR does 2 things:
* Makes it possible to remove from installation leakcanary which should improve development speed a bit in cases when it's not needed
* Removes:
```
        // TODO remove this after we update AGP, it's a bug that has been fixed in 4.1.1 and later
        // see: https://issuetracker.google.com/issues/168794947
        disable "ParcelCreator"
```

It seems to be have been fixed

### Testing instructions

Set in the `gradle.properties`:
* `enableLeakCanary = true` or nothing (should behave the same way. Leakcaray is ON)
* `enableLeakCanary = false` (disables leak canary and does not install it)
(passing `-PenableLeakCanary=false/true` to gradle in command line should behave the same way)


Notice that when `enableLeakCanary = false` then the library is not installed at all 
When it's `true` or you use default then it is installed and it collects leaks
